Release "What's new in Astro - February 2024" (#991)
withastro/astro.build: src/content/blog/whats-new-february-2024.mdx

---
title: "What's new in Astro - February 2024"

> description: "We started of 2024 strong"

Reviewer: ```suggestion
description: "Leap into Astro news with a bonus Februrary day!"
```

publishDate: "February 29, 2024"
authors:
  - elian
coverImage: "/src/content/blog/_images/whats-new-shared.webp"
socialImage: "/src/content/blog/_images/whats-new-february-2024/og.webp"
lang: "en"
---
import ShowcaseGrid from './_whats-new-components/ShowcaseGrid.astro';
import ThemeGrid from './_whats-new-components/ThemeGrid.astro';

little sentence here @todo:

## Updates from Mission Control

- We have a [new channel on Discord](/chat) `#events`, where you can share events happening, share when you're live, and more.

> 

Reviewer: daily dev thing here too?

Author: Yezh! We definitely should!

I was also thinking about a way for ecosystem-ci to be included here?

## Releases

> We're getting things ready for the launch of [Astro Studio](https://studio.astro.build).

Reviewer: ```suggestion
Save the date: [Astro Studio](https://studio.astro.build) launches March 12, and we have a whole week of festivities planned!

While you wait, catch up on Astro and Starlight core releases:
```

- [Astro 4.3](https://astro.build/blog/astro-430/)
- [Astro 4.4](https://astro.build/blog/astro-440/) 
- [Starlight 0.18](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.18.0) 
- [Starlight 0.19](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.19.0)
- [Starlight 0.20](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.20.0)

## Docs

> @sarah to fill out!

Reviewer: ```suggestion
This month Docs added two new guide pages to docs on adding [authentication](https://docs.astro.build/en/guides/authentication/) and [e-commerce](https://docs.astro.build/en/guides/ecommerce/) options to your Astro project! These are two popular features that people ask about, and we wanted to provide some basic guidance to get you started.

Our weekly Discord call “Talking and Doc’ing” is a chance for you to watch Team Docs work in public, jump in a shared Codespace or Gitpod Workspace, and contribute to the docs live, together. This month’s topics included:

- Part two of an epic PR to make docs URL paths consistent with the locations in the sidebar settings that included:
📅 2 weeks.
🕓 4 hours of live streaming.
🧑‍🤝‍🧑 10 people in the same shared codebase at once
🗃 678 files changed.
✅ ALL CHECKS PASSED.

- figuring out what a Starlight `<Steps />` component might look like. Just an ordered list? We think not!

- exploring building in Astro's native i18n routing features to the default Starlight template.
```
Going to commit this, so I can see what the preview looks like!

## Community

### Welcome to our new Maintainers

> Both Fryuni and Oliver have joined Astro as Maintainers!

Reviewer: Add liruifengv, and let's add GitHub links maybe?

Author: Ah great! Maybe better idea to link astro badges?

### Websites

All the community member websites submitted to our Discord `#showcase` channel and featured on community calls this month.

export const sites = [
  { url: 'https://easy-csv.netlify.app/profile', title: '@christopheanfry'},
  { url: 'https://evadecker.com/hypersensitive', title: '@for.eva.new'},
  { url: 'https://warawul.coffee/en/', title: '@chris_origins_coffee'},
  { url: 'https://lindakat-arcade.netlify.app/', title: '@lindakat'},
  { url: 'https://daniell.dev/', title: '@daniellwdb'},
  { url: 'https://heschl.dev/', title: '@joneswastaken'},
  { url: 'https://www.benjamindorey.fr/', title: '@biommxx'},
  { url: 'https://pro-g.eu/', title: '@biommxx'},
  { url: 'https://hitesh.is-a.dev/', title: '@hitesh_dev'},
  { url: 'https://www.tekac.club/humans', title: '@emptybrother7'},
  { url: 'https://evadecker.com/design', title: '@for.eva.new'},
  { url: 'https://joelmturner.com/', title: '@joelmturner'},
  { url: 'https://hermes.cadens.studio/', title: '@cadens.studio'},
  { url: 'https://barvian.me/', title: '@mbarvian'},
  { url: 'https://www.deskpro.com/', title: '@mattheous'},
  { url: 'https://wybm-valentine.netlify.app/', title: '@withered.flowers'},

>   { url: 'https://mike-astrotutorial.netlify.app/', title: '@mikeoxygen'},

Reviewer 2: This one seems to be the Astro tutorial, not sure if expected to be showcased?

Reviewer: Yeah, all Mike did was change his name on the about page... so we can probably skip this one!

  { url: 'https://www.thatisrich.co.uk/', title: '@thatisrich'},
  { url: 'https://wallcreo.com/', title: '@stan.kukucka'},
  { url: 'https://osteopathe-le-thor.fr/', title: '@arnaud_83976'},
  { url: 'https://dan-portoflio-v3.vercel.app/', title: '@rythmlord'},
  { url: 'https://www.xypnox.com/blag/posts/themescura/introducing-themescura/', title: '@xypnox'},
  { url: 'https://photosite-3cw.pages.dev/', title: '@casungo'},
  { url: 'https://sanquirinoestudio.com.ar/', title: '@.wateredplant'},
  { url: 'https://suplovers.co/', title: '@alejostereo'},
  { url: 'https://gitarchived.org/', title: '@thebigbot'},
  { url: 'https://www.arjancodes.com/', title: '@santiago6916'},
  { url: 'https://pdx-food-weeks.vercel.app/', title: '@joelmturner'},
  { url: 'https://www.helmerdavila.com/', title: '@dynamo_dev_99'},
  { url: 'https://lakelimbo.github.io/', title: '@lakelimbo'},
  { url: 'https://aperofrancophone.com/', title: '@marvinkr'},
  { url: 'https://www.karelvo.com/', title: '@karel5743'},
  { url: 'https://anuwong.com/', title: '@anuwong'},
  { url: 'https://k16e.co/', title: '@k16e.co'},
  { url: 'https://wordclues.com/', title: '@plinpod'},

>   { url: 'https://gitter.im/)'},

Reviewer 2: ```suggestion
  { url: 'https://gitter.im/'},
```

Extra `)`

>   { url: 'https://freestuff.dev/)'},

Reviewer 2: ```suggestion
  { url: 'https://freestuff.dev/'},
```

Extra `)`

  { url: 'https://overlayed.dev/', title: '@hacksore'},
  { url: 'https://5pin-talk.ca/', title: '@jbrains_'},
  { url: 'https://upshot.dev/', title: '@khaya_zulu'},
  { url: 'https://prodkt.cloud/', title: '@prodkt'},
  { url: 'https://soulrest.melos.church/', title: '@cameronpak'},
  { url: 'https://designzen.co/', title: '@frank_stallone'},
  { url: 'https://newsletter.howd.dev/posts/01-gus-moxie/', title: '@bonniedotdev'},
  { url: 'https://www.mickeymarse.dev/', title: '@mickeymarse'},
  { url: 'https://mingjun.lu/blog', title: '@mingjunlu'},
  { url: 'https://coderdiaz.com/', title: '@coderdiaz'},
  { url: 'https://streamdal.com/', title: '@dosplatos'},
  { url: 'https://moyadaniel.com/', title: '@rythmlord'},

> ];

Reviewer: I think these are the new ones so far:

for.eva.new stuff from Tuesday:
https://eva.town/projects/boundaries-map
https://eva.town/projects/commonplace

 https://futrloggr.vercel.app/ by xylis_

https://loquierasono.democracia.com.es/ by aitor

<ShowcaseGrid items={sites} />

### Starlight in the wild

Take a peek at the new Starlight sites that we discovered this month.

export const starlightSites = [
{ url: 'https://codegenie.codes/', title: '@brett.codegenie_06005'},
{ url: 'https://sloth-pipe.vercel.app/', title: 'tvswgnr]'},
{ url: 'https://letsar.github.io/state_watcher/', title: 'letsar'},
{ url: 'https://beta.learning.nicholasdwest.com/', title: '@ndw0009'},
{ url: 'https://eddienubes.github.io/sagetest/', title: 'eddienubes'},
{ url: 'https://astro-snipcart.vercel.app/', title: 'lloydjatkinson'},
{ url: 'https://zanobijs.vercel.app/', title: 'devdroide'},
{ url: 'https://vue-simple-dnd.netlify.app/', title: '@florian_lefebvre'},
{ url: 'https://www.waterfox.net/', title: '@aks15501'},
{ url: 'https://thehangedmanpub.netlify.app/', title: '@enigmalea'},

> { url: 'https://vinoth.info/react-sketch-canvas/>', title: 'vinothpandian'},

Reviewer 2: ```suggestion
{ url: 'https://vinoth.info/react-sketch-canvas/', title: 'vinothpandian'},
```

Extra `>`

> { url: 'https://developer.fashionunited.com/>', title: 'FashionUnited'},

Reviewer 2: ```suggestion
{ url: 'https://developer.fashionunited.com/', title: 'FashionUnited'},
```

Extra `>`

## Releases

We're getting things ready for the launch of [Astro Studio](https://studio.astro.build).

- [Astro 4.3](https://astro.build/blog/astro-430/)
- [Astro 4.4](https://astro.build/blog/astro-440/) 
- [Starlight 0.18](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.18.0) 
- [Starlight 0.19](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.19.0)
- [Starlight 0.20](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.20.0)

## Docs

This month Docs added two new guide pages to docs on adding [authentication](https://docs.astro.build/en/guides/authentication/) and [e-commerce](https://docs.astro.build/en/guides/ecommerce/) options to your Astro project! These are two popular features that people ask about, and we wanted to provide some basic guidance to get you started.

Our weekly Discord call “Talking and Doc’ing” is a chance for you to watch Team Docs work in public, jump in a shared Codespace or Gitpod Workspace, and contribute to the docs live, together. This month’s topics included:

> - Part two of an epic PR to make docs URL paths consistent with the locations in the sidebar settings that included:
> 📅 2 weeks.
> 🕓 4 hours of live streaming.
> 🧑‍🤝‍🧑 10 people in the same shared codebase at once
> 🗃 678 files changed.
> ✅ ALL CHECKS PASSED.
> 
> - figuring out what a Starlight `<Steps />` component might look like. Just an ordered list? We think not!
> 
> - exploring building in Astro's native i18n routing features to the default Starlight template.

Reviewer: ```suggestion
- Part two of an epic PR to make docs URL paths consistent with the locations in the sidebar settings that included:
	- 📅 2 weeks.
	- 🕓 4 hours of live streaming.
	- 🧑‍🤝‍🧑 10 people in the same shared codebase at once
	- 🗃 678 files changed.
	- ✅ ALL CHECKS PASSED.

- figuring out what a Starlight `<Steps />` component might look like. Just an ordered list? We think not!

- exploring building in Astro's native i18n routing features to the default Starlight template.
```

> - exploring building in Astro's native i18n routing features to the default Starlight template.

Reviewer: @ElianCodes I will need you to figure out why my bullet points are weird!  :joy:

> - figuring out what a Starlight `<Steps />` component might look like. Just an ordered list? We think not!
> 
> - exploring building in Astro's native i18n routing features to the default Starlight template.

Reviewer: ```suggestion
- Figuring out what a Starlight `<Steps />` component might look like. Just an ordered list? We think not!

- Exploring building in Astro's native i18n routing features to the default Starlight template.
```

- [Resume website with TinaCMS](https://astro-resume02.vercel.app/) by @alpeshnakrani

### Websites

All the community member websites submitted to our Discord `#showcase` channel and featured on community calls this month.

export const sites = [
  { url: 'https://easy-csv.netlify.app/profile', title: '@christopheanfry'},
  { url: 'https://evadecker.com/hypersensitive', title: '@for.eva.new'},
  { url: 'https://warawul.coffee/en/', title: '@chris_origins_coffee'},
  { url: 'https://lindakat-arcade.netlify.app/', title: '@lindakat'},
  { url: 'https://daniell.dev/', title: '@daniellwdb'},
  { url: 'https://heschl.dev/', title: '@joneswastaken'},
  { url: 'https://www.benjamindorey.fr/', title: '@biommxx'},
  { url: 'https://pro-g.eu/', title: '@biommxx'},
  { url: 'https://hitesh.is-a.dev/', title: '@hitesh_dev'},
  { url: 'https://www.tekac.club/humans', title: '@emptybrother7'},
  { url: 'https://evadecker.com/design', title: '@for.eva.new'},
  { url: 'https://joelmturner.com/', title: '@joelmturner'},
  { url: 'https://hermes.cadens.studio/', title: '@cadens.studio'},
  { url: 'https://barvian.me/', title: '@mbarvian'},
  { url: 'https://www.deskpro.com/', title: '@mattheous'},

>   { url: 'https://wybm-valentine.netlify.app/', title: '@withered.flowers'},
>   { url: 'https://mike-astrotutorial.netlify.app/', title: '@mikeoxygen'},
>   { url: 'https://www.thatisrich.co.uk/', title: '@thatisrich'},

Author: ```suggestion
  { url: 'https://wybm-valentine.netlify.app/', title: '@withered.flowers'},
  { url: 'https://www.thatisrich.co.uk/', title: '@thatisrich'},
```

- [Starlight 0.20](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.20.0)

## Docs

This month Docs added two new guide pages to docs on adding [authentication](https://docs.astro.build/en/guides/authentication/) and [e-commerce](https://docs.astro.build/en/guides/ecommerce/) options to your Astro project! These are two popular features that people ask about, and we wanted to provide some basic guidance to get you started.

Our weekly Discord call “Talking and Doc’ing” is a chance for you to watch Team Docs work in public, jump in a shared Codespace or Gitpod Workspace, and contribute to the docs live, together. This month’s topics included:

- Part two of an epic PR to make docs URL paths consistent with the locations in the sidebar settings that included:
	- 📅 2 weeks.
	- 🕓 4 hours of live streaming.
	- 🧑‍🤝‍🧑 10 people in the same shared codebase at once
	- 🗃 678 files changed.
	- ✅ ALL CHECKS PASSED.

- figuring out what a Starlight `<Steps />` component might look like. Just an ordered list? We think not!

- exploring building in Astro's native i18n routing features to the default Starlight template.

## Community

> ### Welcome to our new Maintainers
> 
> Both Fryuni and Oliver have joined Astro as Maintainers!
> 
> Welcome! 🎉

Author: ```suggestion
### Welcome to our new Maintainers

[Fryuni](https://astro.badg.es/contributor/Fryuni/), [Oliver](https://astro.badg.es/contributor/OliverSpeir/), and [liruifengv](https://astro.badg.es/contributor/liruifengv/) have joined Astro as Maintainers!

Welcome! 🎉
```

---
title: "What's new in Astro - February 2024"
description: "Leap into Astro news with a bonus Februrary day!"
publishDate: "February 29, 2024"
authors:
  - elian
coverImage: "/src/content/blog/_images/whats-new-shared.webp"
socialImage: "/src/content/blog/_images/whats-new-february-2024/og.webp"
lang: "en"
---
import ShowcaseGrid from './_whats-new-components/ShowcaseGrid.astro';
import ThemeGrid from './_whats-new-components/ThemeGrid.astro';

Leap into Astro news with a bonus Februrary day!

## Updates from Mission Control

> - We have a [new channel on Discord](https://astro.build/chat) `#events`, where you can share events happening, share when you’re live, and more.
> - We [discussed creating an `ecosystem-ci`](https://discord.com/channels/830184174198718474/1204369341642973185) to provide early warnings and automatic notifications to integration and theme authors about Astro updates including potential breaking changes.
> - We expanded the Astroverse to [daily.dev](https://dly.to/u1jkve3OtCn): updates, releases, showcase items, and more. Another way to get your Astro news and interact with the community. Visit @astrodotbuild and join our squad!

Reviewer: ```suggestion
- We have a [new channel on Discord](https://astro.build/chat) `#events`, where you can share events happening online or in your community, post when you’re going live on a stream to get viewers, and more.
- We [discussed creating an `ecosystem-ci`](https://discord.com/channels/830184174198718474/1204369341642973185) similar to Vite and Volar for better communication about breaking changes in Astro updates with integration and theme authors.
- We expanded the Astroverse to [daily.dev](https://dly.to/u1jkve3OtCn): updates, releases, showcase items, and more. Another way to get your Astro news and interact with the community. Join our squad!
```

- [Adammatthiesen/astro-commercejs](https://github.com/Adammatthiesen/astro-commercejs) by @adamm2047
- [inline-mod-astro](https://inox-tools.vercel.app/inline-mod) by @fryuni
- [JulianCataldo/og-images-generator](https://github.com/JulianCataldo/og-images-generator#astro-integration) by @juliancataldo
- [BryceRussell/astro-pages](https://github.com/BryceRussell/astro-pages) by @bryceguy
- [ktym4a/astro-page-insight](https://github.com/ktym4a/astro-page-insight) by @ktym4a
- [kindspells/astro-sri-scp](https://www.npmjs.com/package/@kindspells/astro-sri-csp) by @andreu
- [Tiny toolbar app to show Tailwind Breakpoints](https://elk.zone/m.webtoo.ls/@zero_gravitas@indieweb.social/111925333428424557)	
- [frankstallone/astro-cube-boilerplate](https://github.com/frankstallone/astro-cube-boilerplate) by @frank_stallone
- [KindSpells/astro-shield](https://github.com/KindSpells/astro-shield) by @castarco
- [natemoo-re/astro-remote](https://github.com/natemoo-re/astro-remote)	by @nmoodev
- [Generate e-mails with Astro](https://github.com/luckydye/astro-email/tree/main) by @luckydye

### Themes & Templates

Several new Astro themes were added & released this month. Try them out!

> - [Illustration based template](https://github.com/manulthanura/Illustration-Based-Portfolio) by @manulthanura
> - [Astroplate](https://astro.build/themes/details/astroplate/) by @tffarhad
> - [Quantum](https://cosmicthemes.com/themes/quantum/) by @lastchaos.
> - [event-edge.gerome.dev](https://event-edge.gerome.dev/) by @gerome.dev
> - [ScrewFast](https://github.com/mearashadowfax/ScrewFast) by @soslan
> - [MinimalStudio, theme built with Astro, TailwindCSS and Alpine](https://minimalstudio.vercel.app/) by @mike_andreuzza
> - [Resume website with TinaCMS](https://astro-resume02.vercel.app/) by @alpeshnakrani

Reviewer 3: ```suggestion
- { url: 'https://github.com/manulthanura/Illustration-Based-Portfolio', title: '@manulthanura'},
- { url: 'https://astro.build/themes/details/astroplate/', title: '@tffarhad'},
- { url: 'https://cosmicthemes.com/themes/quantum/', title: '@lastchaos.'},
- { url: 'https://event-edge.gerome.dev/', title: '@gerome.dev'},
- { url: 'https://screwfast.uk/', title: '@soslan'},
- { url: 'https://minimalstudio.vercel.app/', title: '@mike_andreuzza'},
- { url: 'https://astro-resume02.vercel.app/', title: '@alpeshnakrani'},
```

Reviewer 3: astroplate, quantum, screwfast only have their preview/OG images linked 🤔 maybe this suggestion not good 😞

Reviewer: Sadly yes, this will have to be another PR explicitly targeting the component to address this -- make sure the link goes to the repo but the preview comes from the demo, for example.

---
title: "What's new in Astro - February 2024"

> description: "Leap into Astro news with a bonus Februrary day!"

Reviewer 4: ```suggestion
description: "Leap into Astro news with a bonus February day!"
```

publishDate: "February 29, 2024"
authors:
  - elian
coverImage: "/src/content/blog/_images/whats-new-shared.webp"
socialImage: "/src/content/blog/_images/whats-new-february-2024/og.webp"
lang: "en"
---
import ShowcaseGrid from './_whats-new-components/ShowcaseGrid.astro';
import ThemeGrid from './_whats-new-components/ThemeGrid.astro';

> Leap into Astro news with a bonus Februrary day!

Reviewer 4: ```suggestion
Leap into Astro news with a bonus February day!
```

## Updates from Mission Control

- We have a [new channel on Discord](https://astro.build/chat) `#events`, where you can share events happening, share when you’re live, and more.

> - We started discussing an `ecosystem-ci` conversation on [Discord](/chat)

Reviewer: ```suggestion
- We [discussed creating an `ecosystem-ci`](https://discord.com/channels/830184174198718474/1204369341642973185) to provide early warnings and automatic notifications to integration and theme authors about Astro updates including potential breaking changes.
```

> - We expanded the Astroverse a tiny bit further with the addition of [daily.dev](https://dly.to/u1jkve3OtCn)

Reviewer: ```suggestion
- We expanded the Astroverse to [daily.dev](https://dly.to/u1jkve3OtCn): updates, releases, showcase items, and more. Another way to get your Astro news and updates. Visit @astrodotbuild and join our squad!
```

  { url: 'https://anuwong.com/', title: '@anuwong'},
  { url: 'https://k16e.co/', title: '@k16e.co'},
  { url: 'https://wordclues.com/', title: '@plinpod'},
  { url: 'https://gitter.im/'},
  { url: 'https://freestuff.dev/'},
  { url: 'https://overlayed.dev/', title: '@hacksore'},
  { url: 'https://5pin-talk.ca/', title: '@jbrains_'},
  { url: 'https://upshot.dev/', title: '@khaya_zulu'},
  { url: 'https://prodkt.cloud/', title: '@prodkt'},
  { url: 'https://soulrest.melos.church/', title: '@cameronpak'},
  { url: 'https://designzen.co/', title: '@frank_stallone'},
  { url: 'https://newsletter.howd.dev/posts/01-gus-moxie/', title: '@bonniedotdev'},
  { url: 'https://www.mickeymarse.dev/', title: '@mickeymarse'},
  { url: 'https://mingjun.lu/blog', title: '@mingjunlu'},
  { url: 'https://coderdiaz.com/', title: '@coderdiaz'},
  { url: 'https://streamdal.com/', title: '@dosplatos'},
  { url: 'https://moyadaniel.com/', title: '@rythmlord'},
  { url: 'https://eva.town/projects/boundaries-map', title: "@for.eva.new"},
  { url: 'https://eva.town/projects/commonplace', title: "@for.eva.new"},
  { url: 'https://futrloggr.vercel.app/', title: '@xylis_'},

>   { url: 'loquierasono.democracia.com.es', title: "@aitor"},

Reviewer: ```suggestion
  { url: 'https://loquierasono.democracia.com.es', title: "@aitor"},
```
